pkg/operation/botanist/controlplane.go

```diff
@@ -1049,37 +1053,55 @@ func (b *Botanist) DeployETCD(ctx context.Context) error {
 		hvpaEnabled = gardenletfeatures.FeatureGate.Enabled(features.HVPAForShootedSeed)
 	}
 
-	etcdConfig := map[string]interface{}{
+	defaultValues := map[string]interface{}{
 		"podAnnotations": map[string]interface{}{
-			"checksum/secret-etcd-ca":         b.CheckSums[v1beta1constants.SecretNameCAETCD],
-			"checksum/secret-etcd-server-tls": b.CheckSums["etcd-server-tls"],
-			"checksum/secret-etcd-client-tls": b.CheckSums["etcd-client-tls"],
+			"checksum/secret-etcd-ca":          b.CheckSums[v1beta1constants.SecretNameCAETCD],
+			"checksum/secret-etcd-server-cert": b.CheckSums[common.EtcdServerTLS],
+			"checksum/secret-etcd-client-tls":  b.CheckSums[common.EtcdClientTLS],
 		},
 		"maintenanceWindow": b.Shoot.Info.Spec.Maintenance.TimeWindow,
 		"storageCapacity":   b.Seed.GetValidVolumeSize("10Gi"),
 	}
 
-	etcd, err := b.InjectSeedShootImages(etcdConfig, common.ETCDImageName)
+	values, err := b.InjectSeedShootImages(defaultValues,
+		common.ETCDImageName,
+		common.ETCDBackupRestoreImageName,
+	)
 	if err != nil {
 		return err
 	}
 
+	values["etcd"] = map[string]interface{}{}
+	values["sidecar"] = map[string]interface{}{}
+
 	for _, role := range []string{common.EtcdRoleMain, common.EtcdRoleEvents} {
-		etcd["role"] = role
+		values["role"] = role
+		values["backupEnabled"] = false
+		values["provider"] = ""
 		if role == common.EtcdRoleMain {
 			// etcd-main emits extensive (histogram) metrics
-			etcd["metrics"] = "extensive"
-			etcd["hvpa"] = map[string]interface{}{
+			values["metrics"] = "extensive"
+			values["hvpa"] = map[string]interface{}{
 				"enabled": hvpaEnabled,
 				"minAllowed": map[string]interface{}{
 					"cpu":    "200m",
 					"memory": "700M",
 				},
 			}
+			if b.Seed.Info.Spec.Backup != nil {
+				values["provider"] = b.Seed.Info.Spec.Backup.Provider
+				values["backupEnabled"] = true
+				secret := &corev1.Secret{}
+				if err := b.K8sSeedClient.Client().Get(ctx, kutil.Key(b.Shoot.SeedNamespace, common.BackupSecretName), secret); err != nil {
+					return err
+				}
+				values["prefix"] = common.GenerateBackupEntryName(b.Shoot.Info.Status.TechnicalID, b.Shoot.Info.Status.UID)
+				values["container"] = string(secret.Data[common.BackupBucketName])
```

Comment:
Can you add this value with some default to `values.yaml`?

Reply:
yes.